### ISSUE TITLE
generalize "sanger" email

### ISSUE THREAD

**Issue by robinandeer** [Clinical-Genomics/scout#597]
- [ ] call it something like "verify"
- [ ] support for SV variants
  - [ ] size, coords

**Comment by moonso** [comment]
Would you like to Gove this some thinking @northwestwitch ?

**Comment by northwestwitch** [comment]
Yes but I don't know if I'll have the tine before the next user meeting.

**Comment by northwestwitch** [comment]
What should be done here? Pass info such as chromosome, start and stop and print them in the Sanger email?

**Comment by dnil** [comment]
Yep, that should be it: e.g. ref, alt, subtype of variant as well. And rename to something more general, such as "Verify" or such instead of "Sanger".

**Comment by northwestwitch** [comment]
Ok, I'll work on it soon!

**Comment by northwestwitch** [comment]
Extending the Sanger sequencing to SVs is kind of tricky, because what if they're too big? Should we sequence each of the breakpoints instead?

**Comment by dnil** [comment]
Exactly, more or less. :-) Large on reference, but not necessarily on the affected individual DNA - with a deletion eg. As long as all info on the SV is on the email, the user can determine wether to use eg Sanger, MLPA, array or FISH. Hence verification, rather than Sanger!

**Comment by northwestwitch** [comment]
Ok! But then I didn't grasp the whole thing here, and I have other questions (sorry):

1) Should Sanger functionality for SNVs stay as it is and I should implement a similar but distinct one for the SVs?

2) What do you mean with "Large on reference .."? I should write as more as possible about the SV?
 
3) Should be a multiple choice present for the options you mentioned (Sanger, MLPA, array or FISH)? And should the mail be sent to the same recipients specified in the institute?

**Comment by dnil** [comment]
Don’t ever be sorry for having questions! 😄
My recollection of the discussion is now after some time likely very biased by my own opinions, but here goes:

1. Same content as before for SNVs, but new name.  Verify is nice and general and would most often be done by Sanger for SNVs.

2. I can draw that at some point,  but yeah, details on the whole variant.

3. From our perspective, it being true/confirmed by complementary method is the most important feature. We would have detailed records of how this was done elsewhere. No need to worry about method in scout, methinks. Same recipients!
